cmd/util/util.go

```diff
@@ -220,6 +222,32 @@ minikube config set WantKubectlDownloadMsg false
 	}
 }
 
+func GetCrictlConfigCommand(cfg map[string]string) (string, error) {
+	var (
```

Comment:
unless I'm missing something, there is no need for the var declaration here.

Reply:
Mostly this was just copy-paste from another similar function